### PR TITLE
N36: lyon 1.0

### PR DESCRIPTION
**Pull request by nical**
_Part of #1098_

Here is a section about the lyon 1.0.0 release.

### CODE REVIEW

File: content/news/036/index.md [hunk 1]
```diff
@@ -76,6 +76,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+Lyon made its symbolic [`1.0.0` release](https://crates.io/crates/lyon/1.0.0)
+reflecting the stability of the project. Highlights in this release include:
+ - Initial support for variable line width in the stroke tessellator.
+ - An efficient algorithm to query positions at given distances along a path.
+ - Improved support for specifying custom endpoint attributes in paths and algorithms.
+ - And more. You can read the [announcement blog post here](https://nical.github.io/posts/lyon-1-0.html).
+
+_Discussions: [/r/rust](https://reddit.com/r/rust/comments/vwdxim/announcing_lyon_100),
+[Twitter](https://twitter.com/nicalsilva/status/1546424285442473987?s=20&t=S1fXSoh2zWHbfTImCGYpPQ)_
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:96]:
```suggestion
[Twitter](https://twitter.com/nicalsilva/status/1546424285442473987?s=20&t=S1fXSoh2zWHbfTImCGYpPQ)_

[Lyon]: https://github.com/nical/lyon
```

File: content/news/036/index.md [hunk 2]
```diff
@@ -76,6 +76,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+![Variable width stroke in action](https://nical.github.io/images/variable-width-stroke/lyon-1-0.png)
+
+[Lyon] ([GitHub](https://github.com/nical/lyon)) by [Nical](https://github.com/nical)
+is a collection of crates providing various 2D vector graphics utilities, including
+fast tessellation algorithms, easy to integrate in typical GPU accelerated rendering
+engines.
+
+Lyon made its symbolic [`1.0.0` release](https://crates.io/crates/lyon/1.0.0)
+reflecting the stability of the project. Highlights in this release include:
+ - Initial support for variable line width in the stroke tessellator.
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:90]:
```suggestion
reflecting the stability of the project. Highlights in this release include:

 - Initial support for variable line width in the stroke tessellator.
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Looks good for the most part, just a few formatting tweaks.

Could you also add the image to the repo, rather than using an external link? We prefer to self-host the images in case links go stale in the future.

File: content/news/036/index.md [hunk 3]
```diff
@@ -76,6 +76,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+fast tessellation algorithms, easy to integrate in typical GPU accelerated rendering
+engines.
+
+Lyon made its symbolic [`1.0.0` release](https://crates.io/crates/lyon/1.0.0)
+reflecting the stability of the project. Highlights in this release include:
+
+ - Initial support for variable line width in the stroke tessellator.
+ - An efficient algorithm to query positions at given distances along a path.
+ - Improved support for specifying custom endpoint attributes in paths and algorithms.
+ - And more. You can read the [announcement blog post here](https://nical.github.io/posts/lyon-1-0.html).
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:94]:
```suggestion
- Initial support for variable line width in the stroke tessellator.
- An efficient algorithm to query positions at given distances along a path.
- Improved support for specifying custom endpoint attributes in paths and algorithms.
- And more. You can read the [announcement blog post here](https://nical.github.io/posts/lyon-1-0.html).
```

**Comment by nical** [comment]
Sure, here is a new version with the formatting tweaks and the image.

File: content/news/036/index.md [hunk 4]
```diff
@@ -76,6 +76,27 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+![Variable width stroke in action](lyon.png)
+
+[Lyon] ([GitHub](https://github.com/nical/lyon)) by [Nical](https://github.com/nical)
+is a collection of crates providing various 2D vector graphics utilities, including
+fast tessellation algorithms, easy to integrate in typical GPU accelerated rendering
+engines.
+
+Lyon made its symbolic [`1.0.0` release](https://crates.io/crates/lyon/1.0.0)
+reflecting the stability of the project. Highlights in this release include:
+- Initial support for variable line width in the stroke tessellator.
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:90]:
```suggestion
reflecting the stability of the project. Highlights in this release include:

- Initial support for variable line width in the stroke tessellator.
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: ty!

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Merged, thanks for the PR and congrats on 1.0 :D

**Pull request merged**